Formula/qt-gstreamer.rb

```diff
@@ -0,0 +1,47 @@
+class QtGstreamer < Formula
+  desc "C++ bindings for GStreamer with a Qt-style API"
+  homepage "https://gstreamer.freedesktop.org/modules/qt-gstreamer.html"
+  url "https://gstreamer.freedesktop.org/src/qt-gstreamer/qt-gstreamer-1.2.0.tar.xz"
+  sha256 "9f3b492b74cad9be918e4c4db96df48dab9c012f2ae5667f438b64a4d92e8fd4"
+  head "https://anongit.freedesktop.org/git/gstreamer/qt-gstreamer.git"
+
+  depends_on "boost"
+  depends_on "cmake" => :build
+  depends_on "glib"
+  depends_on "gstreamer"
+  depends_on "gst-plugins-base"
+  depends_on "pkg-config" => :build
```

Comment:
Purely a style nit but you'll make me happier if you stick the `:build` options together 😅.